src/main/java/org/isf/menu/gui/Login.java

```diff
@@ -125,7 +125,7 @@ public void keyReleased(KeyEvent event) {
 	public Login(MainMenu parent) {
 		super(parent, MessageBundle.getMessage("angal.login.title"), true);
 
-		usersListLogin = GeneralData.getGeneralData().getUSERSLISTLOGIN();
+		//usersListLogin = GeneralData.getGeneralData().getUSERSLISTLOGIN();
```

Comment:
Why this instruction has been commented out?

Reply:
Ignore it.